### ISSUE TITLE
Instability warning for let bindings in const fn doesn't list requisite feature flag

### ISSUE THREAD

**Issue by varkor** [rust-lang/rust#57544]
```rust
const fn foo() {
    let _x = 5;
}
```
reports:
```
error: local variables in const fn are unstable
 --> src/lib.rs:2:9
  |
2 |     let _x = 5;
  |         ^^

error: aborting due to previous error
```
This should report to the user which feature flag should be added to use this feature.

**Comment by Centril** [comment]
They are stable on nightly anyways so there might not be anything to fix here?
